flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "skip-icons-check": true
```

Comment:
is this still needed since you add an icon with a patch?

Reply:
The patch is not applied by the Flatpak builder for reasons I don't understand.

Reply:
if you look in the logs:

```
FB: Running: git apply -v -p1 /srv/buildbot/worker/build-x86_64-6/build/icon.patch
Skipped patch 'icons/128-apps-choqok.png'.
Skipped patch 'icons/128-apps-choqok_offline.png'.
Skipped patch 'icons/CMakeLists.txt'.
```

Also I see there is an svgz version of it and it should be usable. At best just gunzip it (not sure how we handle svgz)

Reply:
I wonder if that's not failing because this is not a git checkout but an archive.

Try adding `git-init` to the archive source.

Reply:
That worked! Thanks. Manually rebased the patch to avoid the unreleased changes in 
https://invent.kde.org/network/choqok/-/commit/639acbfc6a957cef816a78c124804135013630d1